database/index.js

```diff
@@ -0,0 +1,69 @@
+const {Sequelize} = require('sequelize');
+const orm = new Sequelize('review', 'student', 'student', {dialect: 'mysql'});
+
+var Reviews = orm.define('reviews', {
+  review_id: {
+    type: Sequelize.INTEGER,
+    primaryKey: true,
+    autoIncrement: true,
+  },
+  product_id: {
+    type: Sequelize.INTEGER,
+  },
+  customer_id: {
+    type: Sequelize.INTEGER,
+  },
+  star_rating: {
+    type: Sequelize.STRING,
+  },
+  comfort: {
+    type: Sequelize.STRING,
+  },
+  quality: {
+    type: Sequelize.STRING,
+  },
+  create_date: {
+    type: Sequelize.DATEONLY,
+  },
+  category: {
+    type: Sequelize.STRING,
+  },
+  comment: {
+    type: Sequelize.TEXT,
+  }
+});
+
+var Products = orm.define('products', {
+  product_id: {
+    type: Sequelize.INTEGER,
+    primaryKey: true,
+    autoIncrement: true,
+  },
+});
+
+var Customers = orm.define('customers', {
+  customer_id: {
+    type: Sequelize.INTEGER,
+    primaryKey: true,
+    autoIncrement: true,
+  },
+  name: {
+    type: Sequelize.STRING,
+  }
+});
+
+Customers.hasMany(Reviews, {
+  foreignKey: {
+    name: 'customer_id',
+    allowNull: false
+  }
+});
+// Reviews.belongsTo(Customers, {foreignKey: 'customer_id'});
```

Comment:
Looking at the implementation of the hasMany and belongs to I am unsure if the second parameter is needed. After looking here https://sequelize.org/master/manual/assocs.html , I saw that their was an options parameter but am unsure why it is needed or if you would need it for this implementation. Just an idea since I see you commented out the belongs too portion because you may have also had some confusion around it.

Reply:
I didn't want to use their default id setup so I used that parameter and this way it becomes clearer for me understand which col is foreign key.